apps/prairielearn/src/lib/socket-server.js

```diff
@@ -1,30 +1,66 @@
 // @ts-check
```

Comment:
Did you want to TypeScript the file on this rewrite?

Reply:
In general I'm in favor of typescriptifying but I'm trying to keep this minimal for now (hence also postponing the other `ioredis` conversion to #8633).